akka-http/src/main/scala/akka/http/javadsl/server/directives/RouteDirectives.scala

```diff
@@ -38,14 +37,29 @@ abstract class RouteDirectives extends RespondWithDirectives {
   /**
    * Java-specific call added so you can chain together multiple alternate routes using comma,
    * rather than having to explicitly call route1.orElse(route2).orElse(route3).
+   * @deprecated Use the `RouteDirectives.routes` method instead.
    */
+  @Deprecated
   @CorrespondsTo("concat")
   @varargs def route(alternatives: Route*): Route = RouteAdapter {
     import akka.http.scaladsl.server.Directives._
 
+    require(alternatives.nonEmpty, "Chaining empty list of routes is illegal.")
+
     alternatives.map(_.delegate).reduce(_ ~ _)
   }
 
+  /**
+   * Java-specific call added so you can chain together multiple alternate routes using comma,
```

Comment:
Let's rephrase at the same time I guess; let's remove `Java-specific call added so you can` and say more like "Used tochain multiple alternate routes ..."